ui/Screen/Project.svelte

```diff
@@ -127,52 +169,56 @@
 </style>
 
 <SidebarLayout dataCy="project-screen">
-  <Remote {store} let:data={project} context="project">
-    <Remote store={revisionsStore} let:data={revisions} context="revisions">
-      <Header.Large
-        name={project.metadata.name}
-        urn={project.shareableEntityIdentifier}
-        description={project.metadata.description}
-        stats={project.stats}>
-        <div slot="left">
-          <div style="display: flex">
-            <div class="revision-selector-wrapper">
-              {#if $currentPeerId}
-                <RevisionSelector
-                  currentPeerId={$currentPeerId}
-                  bind:currentRevision={$currentRevision}
-                  {revisions} />
+  <Remote {store} context="project">
+    {#if project}
```

Comment:
Why do we need this check? I thought that the slot (without name) in `Remote` is only rendered on `Success`.

Reply:
With the old patten (`let:data={project}`), that was the case, but now `project` is supplied by the store itself, which can be `Project | undefined`.

Also, `Success` doesn't necessarily mean there is data - it could be possible in some cases for endpoints to return empty arrays successfully (e.g. the `projects/` endpoint could return `[]` if the user has no projects).